samples/operate-first/Makefile

```diff
@@ -0,0 +1,11 @@
+export RELEASE_NAME=m4d
+export NAMESPACE=m4d-system
+export M4D_CHART_PATH=../../charts/m4d
+export CRD_CHART_PATH=../../charts/m4d-crd
+
+.PHONY: helm-template
+helm-template: 
+	helm template $(RELEASE_NAME) -n $(NAMESPACE) $(M4D_CHART_PATH) | ../../hack/tools/bin/yq e 'select(.kind=="Namespace" or .kind=="MutatingWebhookConfiguration" or .kind=="ValidatingWebhookConfiguration" or .kind=="ClusterRole" or .kind=="ClusterRoleBinding")' - > m4d.yaml
+	helm template $(RELEASE_NAME) -n $(NAMESPACE) $(CRD_CHART_PATH) > m4d-crd.yaml
+	python3 -m pip install -r requirements.txt
```

Comment:
Please move the two python calls to a different helm target like `generate`.

Reply:
The use of python3 binary s environment specific. I usually setup a virtual env and avoid messing with the system python. In all virtual envs and in github actions the binary is just called python (even for 3.x). The header in your python script is also environment specific. My suggestion:
1. Change the header in the python script to `#!/usr/bin/env python` (we don't really use it in the Makefile but just for completion)
2. Change python3 to python
3. On your own machine setup a virtual env when you run it. In the next PR when we create a GitHub action it won't be relevant.